### PR TITLE
Fix indentation and api version for k8s deployment

### PR DESCRIPTION
**Pull request by nareddyt**
Need to check if other files need to be updated.

### CODE REVIEW

File: kubernetes/grpc-bookstore.yaml [hunk 1]
```diff
@@ -31,44 +31,47 @@ spec:
...
+      - name: esp
+        image: gcr.io/endpoints-release/endpoints-runtime:2
+        args: [
+          "--listener_port=9000",
+          "--service=SERVICE_NAME",
+          "--rollout_strategy=managed",
+          "--backend=grpc://127.0.0.1:8000",
+          "--service_account_key=/etc/esp/creds/service-account-creds.json",
+        ]
+        # [END service]
```
Comment (TAOXUY) [inline kubernetes/grpc-bookstore.yaml:65]: Remove this START/END service?

Comment (qiwzhang) [reply kubernetes/grpc-bookstore.yaml:65]: No, there are used in the doc

Comment (TAOXUY) [reply kubernetes/grpc-bookstore.yaml:65]: I have gone through the k8s doc and haven't seen any place mentioning this [START/END service]. Maybe [here](https://cloud.devsite.corp.google.com/endpoints/docs/grpc/get-started-kubernetes#configuring_the_service_name_and_starting_the_service)?

Not sure about how they work for doc but they look confusing in this yaml.

Comment (nareddyt) [reply kubernetes/grpc-bookstore.yaml:65]: That is an example of where they are used. It's needed to show a subsection. I don't think it's too confusing, it's commented out.

**Comment by nareddyt** [comment]
@qiwzhang should I update all docs with the indentation and apiService in this PR?

File: kubernetes/grpc-bookstore.yaml (hunk 1 above)
Comment (TAOXUY) [reply kubernetes/grpc-bookstore.yaml:65]: If this yaml stands alone, this subsection definitely doesn't define a service and if they do, it should include the whole block under `container`.

Maybe change it to be START/END esp

Comment (qiwzhang) [reply kubernetes/grpc-bookstore.yaml:65]: Not, we could not change its name.  It is used in the doc.  Here is the context

{% include "cloud/_docwidgets/_github_include.html" with project="endpoints-samples" region_tag="service" file="kubernetes/grpc-bookstore.yaml" prettyprint_lang="yaml" %}

In its region_tag

Comment (TAOXUY) [reply kubernetes/grpc-bookstore.yaml:65]: ACK

**Pull request merged**